extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -65,3 +104,59 @@ export const renderPageAttachmentUI = (pageEl, attachmentEl) => {
   textEl.textContent = openLabel;
   return openAttachmentEl;
 };
+
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+const renderInlinePageAttachmentUI = (win, pageEl, attachmentEl) => {
+  const openAttachmentEl = buildOpenInlineAttachmentElement(pageEl);
+
+  const openLabelAttr = attachmentEl.getAttribute('cta-text');
+  if (openLabelAttr) {
+    const textEl = win.document.createElement('span');
+    textEl.classList.add("i-amphtml-story-inline-page-attachment-label");
+    const openLabel = openLabelAttr && openLabelAttr.trim();
+    textEl.textContent = openLabel;
+    setImportantStyles(textEl, {
+      'display': 'block',
+    });
+    openAttachmentEl.appendChild(textEl);
+  }
+
+  const openImgAttr = attachmentEl.getAttribute('cta-img');
+
+  const ctaImgEl = openAttachmentEl.querySelector(
+    '.i-amphtml-story-inline-page-attachment-img'
+  );
+
+  setImportantStyles(ctaImgEl, {
+    'background-image': 'url(' + openImgAttr + ')',
+  });
+
+  const openImgAttr2 = attachmentEl.getAttribute('cta-img-2');
+
+  if (openImgAttr2) {
+    const ctaImgEl2 = openAttachmentEl.querySelector(
+      '.i-amphtml-story-inline-page-attachment-img-2'
+    );
+    setImportantStyles(ctaImgEl2, {
+      'display': 'block',
+      'background-image': 'url(' + openImgAttr2 + ')'
+    });
+  }
```

Comment:
Since these are optional we should build them only if they're  specified.
We should always try to keep our dom tree as small as possible.

Reply:
The renderPageAttachmentUI method checks for 'cta-img' bc if there are no images specified, we fallback on the old CTA button design.